src/assets/styles/_colors.scss

```diff
@@ -0,0 +1,44 @@
+:root {
+    --calcite-global-ui-blue: #{$h-bb-060};
+    --calcite-global-ui-blue-hover: #{$h-bb-050};
+    --calcite-global-ui-blue-pressed: #{$h-bb-070};
+    --calcite-global-ui-green: #{$h-gg-060};
+    --calcite-global-ui-green-hover: #{$h-gg-050};
+    --calcite-global-ui-green-pressed: #{$h-gg-070};
+    --calcite-global-ui-yellow: #{$h-yy-060};
+    --calcite-global-ui-yellow-hover: #{$h-yy-050};
+    --calcite-global-ui-yellow-pressed: #{$h-yy-070};
+    --calcite-global-ui-red: #{$h-rr-060};
+    --calcite-global-ui-red-hover: #{$h-rr-050};
+    --calcite-global-ui-red-pressed: #{$h-rr-070};
+    --calcite-global-ui-background: #{$blk-005};
+    --calcite-global-ui-foreground: #{$blk-000};
+    --calcite-global-ui-foreground-hover: #{$blk-010};
+    --calcite-global-ui-foreground-pressed: #{$blk-020};
+    --calcite-global-ui-text-1: #{$blk-220};
+    --calcite-global-ui-text-2:#{$blk-170};
+    --calcite-global-ui-text-3: #{$blk-140};
+
+  }
+  
+  :host([theme="dark"]) {
+    --calcite-global-ui-blue: #00A0FF;
```

Comment:
shouldn't these hard-coded hex's use the `-dark` sass vars?
https://github.com/Esri/calcite-colors/blob/master/colors.scss#L485

Reply:
@bstifle, this is where we would need to update them:
https://github.com/Esri/calcite-colors/blob/master/colors.scss#L485

Reply:
This should now be updated.